tests/test_base.py

```diff
@@ -69,3 +69,35 @@ def test_data_type() -> None:
     assert repr(dt) != ""
     assert dt.get_struct() != {}
     assert dt.get_struct(simplified=True) != {}
+
+
+COLUMN_NUMBER = 10
+
+
+def test_identifier_creation() -> None:
+    """Test basic identifier creation."""
+    ident = astx.Identifier("test_var")
+    assert ident.value == "test_var"
+
+
+def test_identifier_with_location() -> None:
+    """Test identifier with location."""
+    loc = astx.SourceLocation(1, COLUMN_NUMBER)
+    ident_with_loc = astx.Identifier("var2", loc=loc)
+    assert ident_with_loc.value == "var2"
+    assert ident_with_loc.loc.line == 1
```

Comment:
it is not necessary to test **loc**.
but I will not block this PR because of that.